client/src/main/kotlin/no/nav/su/se/bakover/client/ExpiringTokenResponse.kt

```diff
@@ -0,0 +1,28 @@
+package no.nav.su.se.bakover.client
+
+import org.json.JSONObject
+import java.time.LocalDateTime
+
+open class ExpiringTokenResponse(
+    private val json: JSONObject
+) {
+    val accessToken: AccessToken = AccessToken(json.getString("access_token"))
+    private val expiresIn: Int = json.getInt("expires_in")
```

Comment:
Hva slags format kommer `expires_in`i? Millis? I så fall kanskje ikke parse til Int, men heller Long?

Reply:
_The “expires_in” value is the number of seconds that the access token will be valid._
ref: https://www.oauth.com/oauth2-servers/making-authenticated-requests/refreshing-an-access-token/

Reply:
Ait ait, kan sikkert uansett caste til Long, siden man på linja under må ta -20L for å caste det til Long der